src/models/UserMapper.php

```diff
@@ -174,6 +196,10 @@ public function transformResults($results, $verbose)
                     . $row['ID'] . '/hosted/';
                 $list[$key]['talk_comments_uri'] = $base . '/' . $version . '/users/'
                     . $row['ID'] . '/talk_comments/';
+                
+                if ($verbose && isset($this->_request->user_id)) {
```

Comment:
Can it be possible that ```$this->_request->user_id``` is **not** set at this point? If so, what happens in line 169?

Or could the ```can_edit```-flag be set there as well?